.github/workflows/tests.yml

```diff
@@ -66,6 +67,9 @@ jobs:
            composer remove --dev doctrine/annotations
       - name: Run test suite
         run: composer test:unit
+      - name: Run code coverage
+        run: vendor/bin/simple-phpunit --colors=always --coverage-clover coverage.xml
+        if: ${{ matrix.php-version == '8.3' && startsWith(matrix.sf-version, '7.0') }}
```

Comment:
Thanks, but imho it should run on the same testing step instead of testing twice one without coverage and second with. 
Also I think `coverage-text` would be better option unless you plan to add upload step somewhere?

Reply:
I haven't added the upload yet because I'm wondering if it makes sense to generate a coverage report for each test and upload it in one go. Something like this:

https://github.com/doctrine/orm/blob/bfb033fe3c8f567e088802fc3641e6bdd163f7ce/.github/workflows/continuous-integration.yml#L93-L97

and

https://github.com/doctrine/orm/blob/bfb033fe3c8f567e088802fc3641e6bdd163f7ce/.github/workflows/continuous-integration.yml#L319-L342